third-party/components/ApprovalCommentForm/ApprovalCommentForm.scss

```diff
@@ -9,6 +9,10 @@
             width: 100%;
         }
 
+        .label {
+            display: none;
```

Comment:
ah unless we never plan on showing it...

Reply:
There's also no reliable way to hide this label in the way the TextArea component is set up right now :(